pkg/pipeline/GlobalCMCSService.go

```diff
@@ -34,43 +43,63 @@ type GlobalCMCSDto struct {
 	Name       string `json:"name"  validate:"required"`
 	Type       string `json:"type" validate:"oneof=environment volume"`
 	//map of key:value, example: '{ "a" : "b", "c" : "d"}'
```

Comment:
remove commented line

Reply:
this comment gives user idea about sample values which api requires.